content/en/tests/developer_workflows.md

```diff
@@ -122,6 +126,38 @@ You can enable test summaries in pull requests with the following steps:
 
 Comments only appear on pull requests that were opened before the test run and that have run at least one test for an enabled test service.
 
+## Create Jira issues
```

Comment:
Two notes:
1. Maybe we could put this closer to GitHub issues?
2. When I demoed this to my team, the first questions was "does this stay in sync if I change the status of a JIRA issue? And the answer is yes. I think that is quite powerful and we might want to mention it here.